models/iris_obs_avoid/iris_obs_avoid.sdf

```diff
@@ -0,0 +1,56 @@
+<sdf version='1.6'>
+  <model name='iris_obs_avoid'>
+    <include>
+      <uri>model://iris</uri>
+    </include>
+
+    <!-- Only difference to regular iris is this sensor and a MAVLink stream -->
+    <link name="bumper_link">
+      <inertial>
+        <pose>0.01 0.025 0.025 0 0 0</pose>
+        <mass>0.01</mass>
+        <inertia>
+          <ixx>4.15e-6</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>2.407e-6</iyy>
+          <iyz>0</iyz>
+          <izz>2.407e-6</izz>
+        </inertia>
+      </inertial>
+
+      <collision name='bumper_collision'>
+          <pose>0 0 0 0 -0 0</pose>
+          <geometry>
+            <box>
+              <size>0.47 0.47 0.11</size>
+            </box>
+          </geometry>
+      </collision>
+
```

Comment:
Can you bring the bumper sensor into a separate SDF file that could be imported in any model? That would make things more flexible to anyone wanting to use a similar sensor on their simulations.

Also, how hard would it be to find a CAD model for the visuals of the bumper sensor? Not mandatory, but could be interesting to have that bumper showing up on the vehicle visuals, even though this model only serves for benchmarking.

Reply:
I'll put it into a separate model, sure.
But I don't really want a visual, at least not for my use-case. It is meant to be a source of ground-truth information for the benchmarking, rather than an actual sensor for the vehicle. 
Also, the way I'm running these, I have disabled the gazebo GUI anyways... (I'm running hundreds of simulations)